cfme/tests/containers/test_provider_crud.py

```diff
@@ -0,0 +1,49 @@
+import uuid
```

Comment:
This shouldnt be here. Its unrelated to the rest of the PR. You are just adding a file that was removed some time ago by mistake. Please make a separate PR fixing this specific issue of missing CRUD tests.